darglint2/flake8_entry.py

```diff
@@ -74,22 +74,22 @@ def add_options(cls, option_manager):
             '--docstring-style',
             default=defaults.style.name,
             parse_from_config=True,
-            help='Docstring style to use for Darglint',
+            help="Docstring style to use for Darglint2",
         )
 
         option_manager.add_option(
             '--strictness',
             default=defaults.strictness.name,
             parse_from_config=True,
-            help='Strictness level to use for Darglint',
+            help="Strictness level to use for Darglint2",
         )
 
         option_manager.add_option(
             '--darglint-ignore-regex',
```

Comment:
@9dogs, I'm in two minds about whether to rename the `--darglint-ignore-regex` Flake8 option to `--darglint2-ignore-regex`. You opinion?

Reply:
I think if we're changing the name everywhere else, we should change it here as well .

Reply:
Yes, all the more since the configuration file has
```ini
[darglint2]
ignore_regex = ...
```

This rename has now been done.